pkg/commands/compute/deploy.go

```diff
@@ -929,6 +930,17 @@ func constructSetupObjects(
 			Stdout: out,
 		}
 
+		so.objectStores = &setup.KVStores{
+			APIClient:      c.Globals.APIClient,
+			AcceptDefaults: c.Globals.Flags.AcceptDefaults,
+			NonInteractive: c.Globals.Flags.NonInteractive,
+			ServiceID:      serviceID,
+			ServiceVersion: serviceVersion,
+			Setup:          c.Manifest.File.Setup.ObjectStores,
```

Comment:
We're effectively aliasing `[setup.object_stores]` to the kv_stores implementation. Discussed internally, and it was decided this was OK to do as an interim solution.